flowsa/common.py

```diff
@@ -77,7 +77,8 @@ def load_crosswalk(crosswalk_name):
                'sector_name': 'Sector_2012_Names',
                'household': 'Household_SectorCodes',
                'government': 'Government_SectorCodes',
-               'BEA': 'NAICS_to_BEA_Crosswalk'
+               'BEA': 'NAICS_to_BEA_Crosswalk',
+               'BEA_2017': 'NAICS_to_BEA_Crosswalk_2017',
                }
```

Comment:
@catherinebirney this was a temporary solution, I'm sure we can come up with something more robust

Reply:
I'm editing that fxn in my branch - I'm getting rid of that dictionary entirely